### PR TITLE
Ignore dependabot on forks

### PR DESCRIPTION
**Pull request by liyishuai**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
NOROUTE
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [ ] New Route
- [ ] Documentation
  - [ ] CN
  - [ ] EN
- [ ] 全文获取 fulltext
  - [ ] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] 日期和时间 date and time
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

## 说明 / Note
Replacement of #8378

### CODE REVIEW

File: .github/workflows/dependabot-fork.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+jobs:
+  dependabot-fork:
+    runs-on: ubuntu-latest
+    name: Ignore dependabot on forks
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Comment Dependabot PR
+        if: github.repository_owner != 'DIYgod' && github.actor == 'dependabot'
```
Comment (TonyRL) [inline .github/workflows/dependabot-fork.yml:13]:
```suggestion
        if: github.repository_owner != 'DIYgod' && github.actor == 'dependabot[bot]'
```

File: .github/workflows/dependabot-fork.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+on: pull_request
+
+jobs:
+  dependabot-fork:
+    runs-on: ubuntu-latest
+    name: Ignore dependabot on forks
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
```
Comment (TonyRL) [inline .github/workflows/dependabot-fork.yml:10]:
```suggestion
        uses: actions/checkout@v2
```

**Review by TonyRL — CHANGES_REQUESTED** [review]

**Comment by TonyRL** [comment]
FYI, GitHub actions on forks are disabled by default. You need to explicitly enable GitHub actions for this to work.

**Comment by liyishuai** [comment]
Maybe worth documenting it somewhere.

**Pull request merged**